resources/postgres/postgres.go

```diff
@@ -163,3 +197,19 @@ func initDB(pgBinDir, pgDataDir, pgUser string) error {
 
 	return nil
 }
+
+func getBytebaseUser() (int, int, error) {
+	bytebaseUser, err := user.Lookup("bytebase")
+	if err != nil {
+		return 0, 0, fmt.Errorf("Please run Bytebase as non-root user or create a user called `bytebase`")
```

Comment:
Please run Bytebase as non-root user. You can use the following command to create a dedicated `bytebase` user to run the application: RUN addgroup -g 113 -S bytebase && adduser -u 113 -S -G bytebase bytebase